src/Files.Shared/Enums/SortOption.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Files.Shared.Enums
+{
+    public enum SortOption : byte
+    {
+        Name,
+        DateModified,
+        DateCreated,
+        Size,
+        FileType,
+        SyncStatus,
+        FileTag,
+        OriginalFolder,
+        DateDeleted
```

Comment:
Can you add comments like in the group options class?

Reply:
@yaichenbaum Those won't be necessary for this PR, as I didn't author these enums.

Reply:
I understand but we have the opportunity to make an improvement that will take a couple of seconds.